src/components/Person/Person.jsx

```diff
@@ -1 +1,46 @@
-// Write code here
+// filename: Person.jsx
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Person.scss';
+
+export function Person({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) {
+
+  const partner = sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <>
```

Comment:
you have a `<section>` wrapper - you don't need a fragment in such case
you would need it if you have `1+` `top-level html tags` - to encapsulate them.
```suggestion
```